pandas/io/json/json.py

```diff
@@ -408,6 +415,12 @@ def read_json(path_or_buf=None, orient=None, typ='frame', dtype=True,
                 {"index": "row 2", "col 1": "c", "col 2": "d"}]}'
     """
 
+    if orient == 'table':
+        if dtype is None:
```

Comment:
not sure why you actually need to make this False here

Reply:
Because the `dtype` parameter is passed to the constructor of `SeriesParser` or `FrameParser`, which both inherit the constructor from `Parser`. And the `Parser` constructor sets `dtype` to False if it is None (`io.json.json.py`, lines 625 and 626):
```python
        if dtype is None:
            dtype = True
```

Reply:
Because the parameter `dtype` is pased to the constructor of either `SeriesParser` or `FrameParser`, which both inherit from `Parser`. And the `Parser` constructor sets `dtype` to True if it is None (`io.json.json.py`, lines 625-626):
```python
        if dtype is None:
            dtype = True
```